common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -313,6 +316,54 @@ def test_from_xml(self):
             'transcripts': {'uk': 'ukrainian_translation.srt', 'de': 'german_translation.srt'},
         })
 
+    @ddt.data(
+        ('course-v1:test_course+test_num+test_run',
+         '/asset-v1:test_course+test_num+test_run+type@asset+block@test.png'),
+        ('test_course/test_num/test_run', '/c4x/test_course/test_num/asset/test.png')
```

Comment:
To add to that, the naming convention is `org`, `course`, and `run` for the three components of a course key